packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -105,7 +94,7 @@ function isParsable(
 async function fetchDestinations(
   destinationServiceUri: string,
   jwt: string,
-  type: DestinationType,
+  type: 'instance' | 'subaccount',
```

Comment:
[pp] For the `DestinationType` and `ProxyStrategy` you could introduce a dedicated type even if they are not used multiple time. No strong opinion.

Reply:
I added a type for `ProxyStrategy` because it is used as a return type there, but I didn't do it for the `DestinationType` because it really is more of a fancy boolean here. If needed this can be added at any time.